app/src/main/java/exh/md/utils/MdUtil.kt

```diff
@@ -256,5 +256,15 @@ class MdUtil {
             return jsonParser.encodeToString(body)
                 .toRequestBody("application/json".toMediaType())
         }
+
+        fun addAltTitleToDesc(description: String, altTitles: List<String>?): String {
+            return if (altTitles.isNullOrEmpty()) {
+                description
+            } else {
+                val altTitlesDesc = altTitles
+                    .joinToString("\n", "Alternative Titles:\n") { "• $it" }
```

Comment:
Does the "Alternative Titles" text need to be localized in the available MangaDex languages? If yes, how?

Reply:
put it into strings.xml so it can be translated

Reply:
You can do a `Injekt.get<Application>.getString()` to get the translatable string